### DISCUSSION TITLE
[Projects Beta] Is it better to use multiple Projects now in place of Labels?

### DISCUSSION THREAD

**Discussion by hale** [community/community#20555] [Projects and Issues]
**Summary:** We use labels in a variety of ways to categorize and tag Issues. GitHub Projects gives us new abilities to organize Issues, and I wonder if that is a better solution than labels.

---

For example, I might use labels for:

- Prioritization of issues (p:critical, p:major, p:minor)
- Thematic grouping
- Organization of stories into initiatives and/or epics (maybe solved better with [Task Lists](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-task-lists))?
- Type of issue (e.g. bugs)

**One limitation of using labels to tag Issues is when they are shown all at once (as is the case with Project boards), you lose the ability to compare and review issues at-a-glance**. E.g. I can't easily see which use cases are necessary vs nice-to-have on a prioritzation board.

I could ask if it's possible to show only a subset of labels on the board, but that feels like it might be working against the intentions of Projects? 

Instead, would it be better to have a separate Project for each of the 'categories of labels'?  

That doesn't exactly work either, though, because I can't join to other boards and reference (say) a theme column from one board on another list of work-in-progress. (FWIW I've created similar views to this in Airtable via an integration with GitHub, which I've now stopped using because the Table view of Issues gets 80% of the way there natively - and that's fantastic).

Thanks in advance for any advice on how to make better use of Projects/Issues/Labels.

**Comment by justlevine** [comment]
@hale I'm still getting used to the new Projects format, but we've slimmed some labels down and are using others differently then before.

Our labels now serve one of three purposes:
1. Issue Characteristics: Crucially, these are things that can only have a single definition at a time, and more importantly define characteristics about the _issue itself_, and not things related to our project/organizational goals. (I'ts a public repo so the focus is labels that help non-organizational users / contributors filter the repo's issue tab. Things like type (bug, feat, chore, etc), status, (backlog, actionable, in progress, etc), breaking/nonbreaking, good first issue, etc.
Notably what's gone are tags related to priority, effort, value, etc. These only have meaning in the context of a larger strategy, so now they only live as Project fields.
As a bonus, we using an action to map these labels to custom fields when ever the field or label gets changed. 

2. **Tags.** These are more flexible, and are a hybrid of thematic groupings and stories. Unlike the labels/fields used for automatic triage that define the issue themselves, these labels exist to define an issue's  relationship to others. it could be all the issues related to a specific functionality, frontend/vs backend, a usage flow, Pretty much anything, but again its relational to other issues, and still not to its relationship to the project. 
An issue can have multiple of these types of labels, but if an issue has more than two or three, you should ask yourself why (should the issue be broken into several smaller ones, is your PM strategy spread too thin, **should this label be better served as an actual project?** etc).
We can then use one of these label to create a specific `focus` view, which can be used to draft stories or task lists, or as a PM aid prioritizing/planning.

3. **Special flags for users:** Labels like like `hacktober` or `help wanted`. These labels neither describe the issue itself, nor serve a project management need. They solely exist for a user perusing the `Issues` tab.

In total, we've gone from ~120 labels to <40, and most issues have max 6 labels , and our most-labeled issue is 8, down from 17. 

Obviously this all depends on what youre using the project to manage - I used issue triage as an example, because more planning-focused projects dont really as many labels since theyre not doing double duty by improving the `Issues` tab UX. And even still YMMV.

But I _do_ think that regardless of the specific use case, a good starting point is to ask which labels dont convey a characteristic about the _issue_ but rather help with Project Management, and try getting rid of those first.
